src/hooks/useUpdateSourceInputSlotOnMultiviewLayouts.tsx

```diff
@@ -58,15 +66,49 @@ export function useUpdateSourceInputSlotOnMultiviewLayouts(): CallbackHook<
               }
             }
           );
-          return updateLayout({
+
+          // Update the db-multiviews with the new layout-input slot
+          await updateLayout({
             ...singleLayout,
             layout: {
               ...singleLayout.layout,
               views: updated
             }
           });
+
+          const updatedPipelines = production?.production_settings.pipelines;
+          if (updatedPipelines && updatedPipelines[0]) {
```

Comment:
Is it normal here that you only update the first pipeline?

Reply:
Not ideal solution, but atm multiviews are attached to LD-pipeline which is hardcoded into always being at first-index. We probably want to support multiviews for all pipelines, but not possible right now because that would overload servers.